batch.py

```diff
@@ -15,6 +15,7 @@
 
 import click
 import logging
+from csv import Dict
```

Comment:
```suggestion
```

for a few python versions now type hints don't need to be capitalised.

If you were to import a specific type here for the type hints, you'd want to use

```
from typing import Dict
```